packages/gasket-cli/src/scaffold/utils.js

```diff
@@ -86,9 +86,30 @@ function ensureAbsolute(filepath) {
   return path.resolve(process.cwd(), filepath);
 }
 
+
+/**
+ * Parses JSON file or string to assign to context
+ *
+ * @param {CreateContext} context - Create context.
+ * @param {Object} configFlags - flags to read config from
+ * @param {string} configFlags.config - JSON string of config values
+ * @param {string} configFlags.configFile - path to JSON file of config values
+ */
+function readConfig(context, { config, configFile }) {
+  if (config) {
+    const parsedConfig = JSON.parse(config);
+    Object.assign(context, parsedConfig);
+  } else if (configFile) {
+    const parsedConfigFile = require(configFile);
```

Comment:
Wondering if a user might point to a relative file in a way that's not compatible with the path logic for `require`. Perhaps should be something like this instead (I'm not sure how to get the gasket root from the create context, so may need modifications):

```suggestion
    const parsedConfigFile = require(path.resolve(context.gasket.config.root, configFile));
```

Reply:
Yeah good point!